extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -57,6 +57,7 @@ export class PinWidget {
     const href = el.getAttribute('data-pin-href');
     const log = el.getAttribute('data-pin-log');
     if (href) {
+      assertAbsoluteHttpOrHttpsUrl(href);
```

Comment:
Asserting that the href is starting with http/https here. 

Won't allow localhost - so testing 🤷
Also can one still force code in the latter part of the URL?